test-common/flatpak-spawn.c

```diff
@@ -91,6 +91,7 @@ flatpak_install (GFile        *updater_dir,
       { NULL, FLATPAK_BINARY },
       { NULL, "install" },
       { "user", NULL },
+      { "assumeyes", NULL },
```

Comment:
In the commit message this flag is typoed.

Reply:
In what release of Flatpak was this flag added? Do we need to bump a build/test-time dependency?

Reply:
Ah, I see 44db804b58b1a5471e7099357ed6f424486c0f61 already added a dependency on the very newest version.